registrations/templates/message_to_signup.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base_email.html' %}
+{% load i18n %}
+
+{% block content %}
+  <tr>
+    <td class="content-cell" style="padding:0;margin:0;text-align:left;">
+      <div class="content-container" style="padding:0 16px;">
+        <div style="margin: 0px 0px 40px">
+          <p style="color:#1a1a1a;font-size:16px;font-weight:normal;line-height:24px;margin:0px;">{{body |safe}}</p>
+        </div>
+        <div class="cta-button-wrapper" style="margin: 0px 0px 40px">
+          <a href="{{linked_registrations_ui_url}}/fi/registration/{{registration_id}}/enrolment/{{signup_id}}/{{cancellation_code}}/edit" target="_blank" rel="noopener" class="cta-button" style="background:#0000bf;border-radius:0px;border:none;height:56px;text-align:center;text-decoration:none;vertical-align:middle;padding:16px 24px;box-sizing:border-box;display:inline-block;">
+            <span style="color:#ffffff;font-size:16px;font-weight:500;line-height:24px;">{% blocktranslate %}Tarkastele ilmoittautumistasi täällä{% endblocktranslate %}</span>
```

Comment:
Does this actually get translated? 🤔 so if I send this email to a Swedish user and a Finnish user, do they get the email in their own language? Or is this outside of the scope of this PR?

Reply:
There are quite many missing translations for registration in general. This could be solved in separate PR.